features/ftue/impl/src/main/kotlin/io/element/android/features/ftue/impl/sessionverification/FtueSessionVerificationFlowNode.kt

```diff
@@ -37,15 +44,18 @@ class FtueSessionVerificationFlowNode @AssistedInject constructor(
     private val secureBackupEntryPoint: SecureBackupEntryPoint,
 ) : BaseFlowNode<FtueSessionVerificationFlowNode.NavTarget>(
     backstack = BackStack(
-        initialElement = NavTarget.Root(showDeviceVerifiedScreen = false),
+        initialElement = NavTarget.Root,
         savedStateMap = buildContext.savedStateMap,
     ),
     buildContext = buildContext,
     plugins = plugins,
 ) {
     sealed interface NavTarget : Parcelable {
         @Parcelize
-        data class Root(val showDeviceVerifiedScreen: Boolean) : NavTarget
+        data object Root : NavTarget
+
+        @Parcelize
+        data class UseAnotherDevice(val showDeviceVerifiedScreen: Boolean) : NavTarget
```

Comment:
The value for `showDeviceVerifiedScreen` is always `true`, so I guess this parameter can be removed.